lib/features/booking/add_inspection_booking_screen.dart

```diff
@@ -32,13 +33,25 @@ class AddInspectionBookingScreen extends StatelessWidget {
 class AddInspectionBookingBody extends StatelessWidget {
   final String id;
   AddInspectionBookingBody({required this.id, super.key});
-  final ValueNotifier<DateTime> _selectedDate =
-      ValueNotifier(DateTime.parse('2023-09-15'));
+  // initialised starttime only contain Date, no time
+  final ValueNotifier<DateTime?> _selectedStartTime =
+      ValueNotifier(DateTime.now().copyWith(hour: 9, minute: 0));
   final ValueNotifier<int> noteChars = ValueNotifier(0);
   final maxChar = 400;
 
+  // To generate list of number for time picker
+  List<int> generateNumbersList(int start, int end) {
+    List<int> numbersList = [];
+    for (int i = start; i <= end; i++) {
+      numbersList.add(i);
+    }
+    return numbersList;
+  }
+
   @override
   Widget build(BuildContext context) {
+    final minutesList = generateNumbersList(0, 59);
```

Comment:
Better to convert this widget to be stateful widget and do this calculation in initState, we also need to dispose all ValueNotifier in dispose()